### DISCUSSION TITLE
Backtick disabled on Vs Code on italian layout when copilot is installed.

### DISCUSSION THREAD

**Discussion by superandrew** [community/community#7712] [Copilot Conversations]
Provided that I installed copilot even if I don't still have access to it (please, let me in 😂), by doing some investigation around https://github.com/microsoft/vscode/issues/136580, I noticed that copilot enabled a keybinding (alt+) which on italian layout on Mac is used to trigger the key ` (backtick).

So installing copilot extension basically breaks the backtick on vs code alone. Not sure what to suggest but I'm pretty sure italian mac devs would be disappointed to switch to alt+9 for the backtick in vs code alone.

**Comment by ptkdev** [comment]
Same problem here. Italian layout - macos

**Comment by AliShadman95** [comment]
Same problem here with macos and italian layout

**Comment by emavitta** [comment]
same as well

**Comment by simonecorsi** [comment]
Still happening here, has anyone found a way out?

**Comment by ptkdev** [comment]
Workaround:
I removed from `package.json` in `~/.vscode/extensions/github.copilot-1.XX.XXXXX/`

This lines:

```
	{
		"command": "editor.action.inlineSuggest.trigger",
		"key": "alt+\\",
		"when": "editorTextFocus && !editorHasSelection && config.github.copilot.inlineSuggest.enable && !inlineSuggestionsVisible"
	}
```

You need to delete it again after each update.

**Comment by simonecorsi** [comment]
TBH I got used in these weeks just using `alt+9`(opt on mac), or preselecting the text and using old keybind, as the definition posted above imply! Just a bit difficult removing an old habit the first days!

In retrospect, I think I'm using more backticks than copilot suggestions, so I may just disable it!

**Comment by nicolopadovan** [comment]
Same problem on Italian keyboard layout.

**Comment by erbanku** [comment]
Same for me, it seems the problem was caused by me accidentally. Removing the below line from `keybindings.json` solved the issue. 😊
```
    {
        "key": "oem_3 1",
        "command": "workbench.files.action.collapseExplorerFolders"
    },
```
![image](https://user-images.githubusercontent.com/79493862/215276144-1cc15bcc-0399-4fb3-a71c-39115c15180b.png)

**Comment by giu1io** [comment]
My workaround (should be permanent):

Go to Keyboard Shortcuts settings in VSCode, search for "trigger inline suggestion", change the CoPilot shortcut to a different shortcuts then `alt + \`

![image](https://github.com/community/community/assets/6680957/1bbff9cd-dbe8-4305-abd4-3a3e2ebb32ed)

**Reply by gianfrancodp** [reply]
What a perfect workaround! Thank you.
